package.json

```diff
@@ -8,59 +8,64 @@
     "ui toolkit",
     "v1"
   ],
-  "main": "dist/scripts/helix-ui.js",
-  "module": "dist/scripts/helix-ui.es.js",
+  "main": "dist/js/helix-ui.cjs.js",
+  "module": "dist/js/helix-ui.esm.js",
+  "unpkg": "dist/js/helix-ui.js",
```

Comment:
Per unpkg.com docs...

> If you omit the file path (i.e. use a “bare” URL), unpkg will serve the file specified by the `unpkg` field in `package.json`, or fall back to `main`.

So, this field was added to serve the UMD bundle if folks load it via...

```html
<!-- with this configuration... -->
<script src="https://unpkg.com/helix-ui"></script>

<!-- unpkg will expand to... -->
<script src="https://unpkg.com/helix-ui@x.y.z[-rc.N]/dist/js/helix-ui.js"></script>
```